examples/mnist/export_test.py

```diff
@@ -0,0 +1,64 @@
+import tensorflow as tf
+import tensorflow_hub as hub
+import unittest
+import tempfile
+import shutil
+import os
+from absl.testing import absltest
+import export
+from export import train_and_export
+
+TMPDIR = tempfile.mkdtemp()
+model = None
+TEST_DATA_DIR = "examples/mnist/test_data"
+
+def load_model(path):
```

Comment:
Please remove this and just use hub.load(path)

Reply:
I'm using this avoid reloading the model everytime. This would reduce the test time. at first I did `hub.load()` in `def setUp(...)` but turns out, at the beginning of each and every step, all the instance variables are cleared and `setUp` is called. So, it is reloading the same model every single time. That's reducing the speed of the test.